src/routes/LoginPrompt/index.jsx

```diff
@@ -27,6 +27,7 @@ const LoginPrompt = ({
 
   useEffect(() => {
     if (isLoggedIn) {
+      // What should be the change here?
       navigate(nextPageUrl || "/self-service/branding");
```

Comment:
@brooketopcoder Please refer to the PR description. But the question in short is do we need to change the url here?

Reply:
@hentrymartin I think we can redirect back to the work dashboard if the user is already logged in.